ihatemoney/models.py

```diff
@@ -175,8 +175,8 @@ def get_bills(self):
             .filter(Bill.payer_id == Person.id)\
             .filter(Person.project_id == Project.id)\
             .filter(Project.id == self.id)\
-            .order_by(Bill.creation_date.desc())\
             .order_by(Bill.date.desc())\
+            .order_by(Bill.creation_date.desc())\
```

Comment:
Why moving this line?

Reply:
to sort by date first... like in #418 .... 
feels a bit weird to me that creation date is the sort col too.

Reply:
Interesting. I wonder if we shouldn't just remove this order_by? Maybe @Phyks has an idea on the matter?

Reply:
As we only store dates we might want to have another sort column....
But indeed, id could serve this purpose.... Maybe the creation date was there to be a bit more explicit or smth